packages/augur-sdk/src/api/ZeroX.ts

```diff
@@ -179,14 +180,34 @@ export class ZeroX {
   }
 
   async getOrders(): Promise<OrderInfo[]> {
-    // TODO when browser mesh supports this back out to using it if _rpc not provided
-    if (!this.rpc) {
-      throw Error('getOrders is not supported on browser mesh');
+    var response;
+    if (this.rpc) {
```

Comment:
Is the intention to have browser mesh just be a fallback for this?

Reply:
yeah